README.md

```diff
@@ -38,15 +37,11 @@ To properly use Draft to containerize your application, create manifest files, a
 
```

Comment:
The list of commands is outdated 

`Draft Create Deployment Files` was split into `Draft a Dockerfile` and `Draft a Kubernetes Deployment and Service`